From e723694849834a9cd104518a61a77bc302749955 Mon Sep 17 00:00:00 2001
From: Marc Mutz <marc.mutz@qt.io>
Date: Wed, 4 May 2022 11:36:48 +0200
Subject: [PATCH 105/157] QBuffer: add missing <limits> include

Amends 4bc85b9850303fa20206f8774af88d72593d3454.

Pick-to: 6.3 6.2 5.15
Task-number: QTBUG-102274
Change-Id: If826043f01155f9854cc69079a09f1b50b47994c
Reviewed-by: Mikko Gronoff <mikko.gronoff@qt.io>
(cherry picked from commit 8ece12e466f8a0c6b0daeb5ed6601094fd7a0dbc)
---
 src/corelib/io/qbuffer.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/corelib/io/qbuffer.cpp b/src/corelib/io/qbuffer.cpp
index 5d9570957b..032c0ddd4b 100644
--- a/src/corelib/io/qbuffer.cpp
+++ b/src/corelib/io/qbuffer.cpp
@@ -41,6 +41,8 @@
 #include <QtCore/qmetaobject.h>
 #include "private/qiodevice_p.h"
 
+#include <limits>
+
 QT_BEGIN_NAMESPACE
 
 /** QBufferPrivate **/
-- 
2.38.1

